hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/FindTargetStrategy.java

```diff
@@ -70,5 +70,5 @@ void reInitialize(List<DatanodeUsageInfo> potentialDataNodes,
    */
   void resetPotentialTargets(@Nonnull Collection<DatanodeDetails> targets);
 
-  Map<DatanodeDetails, Long> getSizeEnteringNodes();
+  ConcurrentHashMap<DatanodeDetails, Long> getSizeEnteringNodes();
```

Comment:
Need use Map<> itself, change all place. But object created can be ConcurrentHashMap  instead of HashMap